composer.json

```diff
@@ -58,7 +58,7 @@
     "prestashop/gridhtml": "^2",
     "prestashop/gsitemap": "^4",
     "prestashop/pagesnotfound": "^2",
-    "prestashop/productcomments": "^4.0",
+    "prestashop/productcomments": "^5.0",
```

Comment:
Is the module updated on purpose or it's a false manipulation with composer? The scope is supposed to be for smarty only no?

I prefer to check because it's supposed to be a backport of https://github.com/PrestaShop/PrestaShop/pull/27337 which was strictly a backport

Reply:
Look at the commit list 😉  this is a commit added by @atomiix to fix the CI https://github.com/PrestaShop/PrestaShop/pull/27350/commits/270e13a14da16d8c54bf01b0c69f1d476e11c2e5